kleinanzeigen_bot/extract.py

```diff
@@ -41,20 +42,16 @@ def extract_special_attributes_from_ad_page(self) -> dict:
 
         :return: a dictionary (possibly empty) where the keys are the attribute names, mapped to their values
         """
-
-        try:
-            details_box = self.driver.find_element(By.CSS_SELECTOR, '#viewad-details')
-            details_list = details_box.find_element(By.XPATH, './/ul')
-            list_items = details_list.find_elements(By.TAG_NAME, 'li')
-            details = {}
-            for list_item in list_items:
-                detail_key = list_item.text.split('\n')[0]
-                detail_value = list_item.find_element(By.TAG_NAME, 'span').text
-                details[detail_key] = detail_value
-
-            return details
-        except NoSuchElementException:
-            return {}
+        belen_conf = self.driver.execute_script("return window.BelenConf")
+        special_attributes_str = belen_conf["universalAnalyticsOpts"]["dimensions"]["dimension108"]
+        special_attributes = json.loads(special_attributes_str)
+        if not isinstance(special_attributes, dict):
+            raise ValueError(
+                "Failed to parse special attributes from ad page."
+                f"Expected a dictionary, but got a {type(special_attributes)}"
+            )
+        special_attributes = {k: v for k, v in special_attributes.items() if not k.endswith('.versand_s')}
```

Comment:
From what I've seen in examples, .versand_s was indeed so far the only one we need to remove from special_attributes.